tests/test_clos/test_cromwell_switch.py

```diff
@@ -0,0 +1,79 @@
+"""Pytests for function Cloudos.cromwell_switch"""
+import json
+import mock
+import pytest
+import requests
+import responses
+from responses import matchers
+from cloudos.clos import Cloudos
+from cloudos.utils.errors import BadRequestException
+from tests.functions_for_pytest import load_json_file
+
+INPUT = "tests/test_data/process_job_list_initial_json.json"
+APIKEY = 'vnoiweur89u2ongs'
+CLOUDOS_URL = 'http://cloudos.lifebit.ai'
+WORKSPACE_ID = 'lv89ufc838sdig'
+CROMWELL_ACTION = "stop"
+
+
+@mock.patch('cloudos.clos', mock.MagicMock())
+@responses.activate
+def test_cromwell_switch():
+    """
+    Test 'get_cromwell_switch' to work as intended
+    API request is mocked and replicated with json files
+    """
+    create_json = load_json_file(INPUT)
```

Comment:
Instead of using a previous file, which could be misleading for future review, just make explicit that we are not really testing the response at all doing something like:
```suggestion
    create_json = json.dumps({"mock": "response"})
```